### PR TITLE
extend image attribute: show as picture

### PR DESCRIPTION
**Pull request by xaylpq**

### CODE REVIEW

File: src/form/editors/impl/document/views/ImageBubbleItemView.ts [hunk 1]
```diff
@@ -88,7 +90,7 @@ export default Marionette.View.extend({
                 allowDelete: this.allowDelete,
                 editorHasHistory: this.editorHasHistory
             },
-            panelPosition: 'right',
+            panelPosition: 'down',
```
Comment (stasguryev) [inline src/form/editors/impl/document/views/ImageBubbleItemView.ts]: можно не указывать

File: src/form/editors/impl/document/templates/imageBubbleItem.html [hunk 2]
```diff
@@ -1,3 +1,26 @@
+{{#if showAsPicture}}
```
Comment (stasguryev) [inline src/form/editors/impl/document/templates/imageBubbleItem.html]: много копипасты

File: src/form/editors/ImageEditorView.js [hunk 3]
```diff
@@ -555,12 +557,13 @@ export default formRepository.editors.Image = BaseCollectionEditorView.extend({
         const childViews = documentElements;
         const length = this.collection.length;
         // invisible children
-        for (let i = MAX_NUMBER_VISIBLE_DOCS; i < length; i++) {
+        const maxNumbers = this.options.showAsPicture ? 1 : MAX_NUMBER_VISIBLE_DOCS;
+        for (let i = maxNumbers; i < length; i++) {
             childViews[i].style.display = 'none';
         }
-        if (length - MAX_NUMBER_VISIBLE_DOCS > 0) {
+        if (length - maxNumbers > 0) {
```
Comment (stasguryev) [inline src/form/editors/ImageEditorView.js]: options?

File: src/form/editors/ImageEditorView.js [hunk 4]
```diff
@@ -31,6 +31,7 @@ const MultiselectAddButtonView = Marionette.View.extend({
 
 const defaultOptions = options => ({
```
Comment (stasguryev) [inline src/form/editors/ImageEditorView.js:27]: MultiselectAddButtonView file

File: resources/styles/documents.css [hunk 5]
```diff
@@ -96,6 +105,16 @@ a.image-buttons__link:hover {
...
 }
 
+.picture_container {
+    position: relative;
+    overflow: hidden;
+    max-width: 100%;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--black);
```
Comment (stasguryev) [inline resources/styles/documents.css]: ?

File: resources/styles/documents.css [hunk 6]
```diff
@@ -35,17 +35,26 @@ a.image-buttons__link:hover {
     align-items: center;
     justify-content: flex-start;
     flex: 1 0 auto;
-    max-width: 400px;
```
Comment (stasguryev) [inline resources/styles/documents.css:38]: screens

Comment (xaylpq) [reply resources/styles/documents.css:38]: Property not used

File: src/form/editors/ImageEditorView.js (hunk 4 above)
Comment (xaylpq) [reply src/form/editors/ImageEditorView.js:27]: Done

File: resources/styles/documents.css (hunk 5 above)
Comment (xaylpq) [reply resources/styles/documents.css]: Done

File: src/form/editors/ImageEditorView.js (hunk 3 above)
Comment (xaylpq) [reply src/form/editors/ImageEditorView.js]: Done

File: src/form/editors/impl/document/templates/imageBubbleItem.html (hunk 2 above)
Comment (xaylpq) [reply src/form/editors/impl/document/templates/imageBubbleItem.html]: Done

File: src/form/editors/impl/document/views/ImageBubbleItemView.ts (hunk 1 above)
Comment (xaylpq) [reply src/form/editors/impl/document/views/ImageBubbleItemView.ts]: Done

**Pull request merged**